### ISSUE TITLE
Total Size of the Dataset

### ISSUE THREAD

**Issue by Kou-99** [HengLan/VastTrack#6]
Hi, thanks for your great work. Given that the dataset is quite large, can you please provide the total size of the training and testing split, both compressed and unpressed? This will help me ensure I have enough disk space to download and extract the files. Thanks in advance for your help!

**Comment by wuhengliangliang** [comment]
Thank you very much for your recognition of our work. Below, we can answer your question:
The size of the test set after decompression is 71g
The size of the test set after compression is 70g
The size of the training set after compression is 686g
The size of the training set before compression is 690g
